packages/api/pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[project]
+name = "wasmer"
+version = "1.1.0"
+description = "Python extension to run WebAssembly binaries"
+readme = "README.md"
+repository = "https://github.com/wasmerio/wasmer-python"
+keywords = ["python", "extension", "webassembly"]
+categories = ["wasm"]
+build = "build.rs"
+
+[package]
+edition = "2018"
+
+[build-system]
+requires = ["maturin>=0.14,<0.15"]
+build-backend = "maturin"
+
+[tool.maturin.metadata]
+category = ["Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Rust",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Compilers",
+    "Topic :: Software Development :: Interpreters",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent"]
+
+
+[tool.maturin.project-url]
+"Source Code" = "https://github.com/wasmerio/wasmer-python/"
+"Bug Tracker" = "https://github.com/wasmerio/wasmer-python/issues"
+"Documentation" = "https://github.com/wasmerio/wasmer-python/"
```

Comment:
Keep in mind that this means we'll forever need to keep `Cargo.toml` and `pyproject.toml` in sync. I'd much prefer things like keywords, version number, description, and so on are specified in `Cargo.toml` so we only need to manage one file per package.

This also makes releases more annoying because release tools like `cargo release` won't know to update the version number in both files.

Reply:
I've this this issue by removing the `project` section from `pyproject.toml`. This means the version will only need to be updated in Cargo.toml.